forc/src/cli/commands/plugins.rs

```diff
@@ -18,15 +18,32 @@ pub struct Command {
     describe: bool,
 }
 
+fn get_file_name(path: &Path) -> String {
+    if let Some(Some(path_str)) = path.file_name().map(|path_str| path_str.to_str()) {
+        path_str.to_owned()
+    } else {
+        path.display().to_string()
+    }
+}
+
 pub(crate) fn exec(command: PluginsCommand) -> ForcResult<()> {
     let PluginsCommand {
         print_full_path,
         describe,
     } = command;
 
+    let mut plugins = crate::cli::plugin::find_all()
+        .map(|path| {
+            print_plugin(path.clone(), print_full_path, describe)
+                .map(|info| (get_file_name(&path), info))
+        })
+        .collect::<Result<Vec<(String, String)>, _>>()?;
+    plugins.sort_by(|a, b| a.0.cmp(&b.0));
+    plugins.dedup_by(|a, b| a.0 == b.0);
```

Comment:
Discussed offline: we'll show all plugins with the full path. When there's a duplicate, we'll indicate which one will be used based on the $PATH